common/compression/stuffit.h

```diff
@@ -52,8 +52,9 @@ class SeekableReadStream;
  *
  * May return 0 in case of a failure.
  */
-Archive *createStuffItArchive(const String &fileName);
-Archive *createStuffItArchive(SeekableReadStream *stream);
+Archive *createStuffItArchive(const String &fileName, bool respectSizeField = false);
+Archive *
+createStuffItArchive(SeekableReadStream *stream, bool respectSizeField = false);
```

Comment:
Is this expected to be in a new line?

Reply:
Fixed. On second thoughts we should never need to seek past the declared end of archive. So I removed the flag altogether